src/lib/server/xlat_eval.c

```diff
@@ -285,23 +285,23 @@ static xlat_action_t xlat_eval_one_letter(TALLOC_CTX *ctx, fr_cursor_t *out, REQ
 
 	case 'c': /* Current epoch time seconds */
 	{
-		struct timeval now;
+		fr_time_t now;
 
-		gettimeofday(&now, NULL);
+		now = fr_time();
 
 		MEM(value = fr_value_box_alloc(ctx, FR_TYPE_UINT64, NULL, false));
-		value->datum.uint64 = (uint64_t)now.tv_sec;
+		value->datum.uint64 = (uint64_t)now;
```

Comment:
`fr_time()` is time in nanoseconds since the server started.  So you need to convert nanoseconds to seconds, and also add in the seconds since the server started.

Reply:
There's a function for that ``fr_time_to_sec``, because this is a very common pattern in the server.